### ISSUE TITLE
memory leak on self._create_readonly += relations_includes

### ISSUE THREAD

**Issue by qianyi3210** [maxtepkeev/python-redmine#257]
redminelib/resources/base.py
128   self._create_readonly += relations_includes
129  self._update_readonly += relations_includes

tools shows memory leak on these two lines,  any suggestion?

**Comment by maxtepkeev** [comment]
@qianyi3210 which tools ?

**Comment by qianyi3210** [comment]
tracemalloc
mycode is as below:

```
class RedmineService(object):
    def __init__(self):
         # self.logger = logger
         self.client = Redmine(_conf.get("redmine", "rm_url"),
                              username=_conf.get("redmine", "rm_username"),
                              password=_conf.get("redmine", "rm_password"),
                              requests={"verify": False})

    def is_issue_exist(self, project_id, subject):
        issues = self.client.issue.filter(project_id=project_id, status_id="open")
        for issue in issues:
            if subject == issue.subject:
                return True
        else:
            return False

redmine_service = RedmineService()
```

**Comment by maxtepkeev** [comment]
Good find, thanks. Fixed in 5b8d4c4.
